### DISCUSSION TITLE
Which port to use, 8000 or 8080?

### DISCUSSION THREAD

**Discussion by hudak** [caronc/apprise-api#180] [Q&A] [answered]
I noticed that the docker image is listening on both 8000 and 8080. Which one is preferred? 8080 seems to be running though an nginx proxy

https://github.com/caronc/apprise-api/blob/d33c50f4669184577b5b983ae3eb92aacd498566/apprise_api/etc/nginx.conf#L52

**Comment by caronc** [comment, marked as answer]
8000 is the port to use if you want to leverage the website to help with the configuration (static file hosting, workers, threading, container log setup, etc).

8080 is the gunicorn application in the background. It's direct to the API (Python CGI interface). I mean, there is no harm using this path if you adopt some of the configuration in the NGinX mapping described above. Personally I would advise using port 8000 as documented though.
